src/boards/include/boards/adafruit_qtpy_rp2040.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+// -----------------------------------------------------
+// NOTE: THIS HEADER IS ALSO INCLUDED BY ASSEMBLER SO
+//       SHOULD ONLY CONSIST OF PREPROCESSOR DIRECTIVES
+// -----------------------------------------------------
+
+#ifndef _BOARDS_ADAFRUIT_QTPY_RP2040_H
+#define _BOARDS_ADAFRUIT_QTPY_RP2040_H
+
+// For board detection
+#define ADAFRUIT_QTPY_RP2040
+
+//------------- UART -------------//
+#ifndef PICO_DEFAULT_UART
+#define PICO_DEFAULT_UART 0
```

Comment:
Mistake in the header here -- these GPIOs are both for UART1.

Reply:
Ahhh, so perhaps https://github.com/raspberrypi/pico-sdk/pull/192#issuecomment-786318489 wasn't such a crazy idea after all :wink:   I'll create an issue for it.

Reply:
fixed